src/tools/schema2kotlin.ts

```diff
@@ -45,11 +45,12 @@ export class Schema2Kotlin extends Schema2Base {
   }
 
   fileHeader(outName: string): string {
+    if (!this.namespace) throw new Error('WHOA!');
```

Comment:
Better error message?

Reply:
haha, this should not be here, thanks for catching. It's verified in the schema2base, removing!